lib/vrl/stdlib/src/parse_key_value.rs

```diff
@@ -422,6 +457,13 @@ fn parse_key<'a, E: ParseError<&'a str> + ContextError<&'a str>>(
     }
 }
 
+fn type_def() -> TypeDef {
+    TypeDef::object(Collection::from_unknown(
+        Kind::boolean() | Kind::bytes() | Kind::array(Collection::from_unknown(Kind::bytes())),
```

Comment:
Thanks for fixing this 👍 

cc/ @fuchsnj or @JeanMertz to validate it since I've been wrong on type defs before.

Reply:
This is correct.